templates/windows/project.nuspec.erb

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id><%= @name %></id>
+    <version><%= "#{@version}.#{@release}" %></version>
```

Comment:
As long as version is `x.y.z` (or `x` or `x.y`) this will work. It only has `x.y.z.a` available and all need to be numerical.